src/aggregate/aggregate_request.c

```diff
@@ -657,13 +651,8 @@ int AREQ_Compile(AREQ *req, RedisModuleString **argv, int argc, QueryError *stat
         goto error;
       }
     } else {
-      rv = handleCommonArgs(req, &ac, status, 0);
-      if (rv == ARG_ERROR) {
-        goto error;
-      } else if (rv == ARG_UNKNOWN) {
-        QueryError_FmtUnknownArg(status, &ac, "<main>");
-        goto error;
-      }
+      QueryError_FmtUnknownArg(status, &ac, "<main>");
```

Comment:
@mnunberg we are calling handleCommonArgs above in the begging of the look, if it would return ERROR or HANDLED we will no reach here, if it reached here it means arg unknown, So I left the code for ARG_UNKNOWN only.